bedrock/cms/management/commands/scrub_exported_cms_data.py

```diff
@@ -0,0 +1,46 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+from django.contrib.auth.models import User
+from django.contrib.sessions.models import Session
+from django.core.management.base import BaseCommand
+
+from wagtail.models import Revision
+from wagtail_localize.models import TranslationSource
+
+from bedrock.utils.management.decorators import alert_sentry_on_exception
+
+
+@alert_sentry_on_exception
+class Command(BaseCommand):
+    """Purge Wagtail-managed data that we don't need in the published site
+    which may be either confusing, outdated or too early to publicly share (as a draft)
+    """
+
+    def add_arguments(self, parser):
+        parser.add_argument("-q", "--quiet", action="store_true", dest="quiet", default=False, help="If no error occurs, swallow all output.")
+
+    def output(self, *args):
+        if not self.quiet:
+            print(*args)
+
+    def handle(self, *args, **options):
+        self.quiet = options["quiet"]
+
+        user_results = User.objects.all().delete()
```

Comment:
Does this break content at all if this cascades out to content the user's created? I don't know the db models but just curious about breaking FKs.

Reply:
True. Maybe we just replace emails with random noise to anonymize.

Reply:
In my experience, it doesn't cascade but sets null - pages can survive even if the FK etc has been zapped - e.g.

https://github.com/wagtail/wagtail/blob/main/wagtail/models/__init__.py#L2849

Reply:
Might be better in the bigger picture to write nonsense @example.com emails, though - not for Wagtail but in case over time we start keying other things to Users that don't degrade as gracefully. Will have a think, as that might be unlikely to happen IRL

Reply:
I'm gonna leave this as-is for now and circle back if we have issue when there's real (but still test-pages) data flowing around